src/app/frontend/common/components/objectmeta/template.html

```diff
@@ -71,6 +71,16 @@
              i18n>UID</div>
         <div value>{{objectMeta?.uid}}</div>
       </kd-property>
+      <kd-property *ngIf="objectMeta?.ownerReferences"
+                   [ngClass]="'object-meta-ownerReferences'">
+        <div key
+             i18n>Owner</div>
+        <div value
+             *ngFor="let ownerReference of objectMeta?.ownerReferences">
```

Comment:
Handling for multiple owner refs is implemented here but I wonder how the view will look like if there will be more than one owner. Can you paste some screenshot?

Reply:
I'd maybe rely on the `controlled by` card component that we use i.e. on pod details and try to reuse it for other resources.

Reply:
Hi @maciaszczykm,

From my understanding, one can't automatically get two owners, but one can manually set them. This is how you get two of them.

I did it out of curiosity and I got this:
![image](https://user-images.githubusercontent.com/297498/125077869-fd57b800-e097-11eb-9b6c-a61dfd8a6b15.png)

But I think @floreks is right. It's better to ditch this and just put the `controlled by` card everywhere.

Reply:
I am thinking again. Do we really need the controller card ? Currently it's just used on the pods page, which has already too much content. Maybe removing it would be better. Just thinking outloud.

Reply:
I'd still keep it but focus on reorganizing information on the detail pages overall. Use tabs, maybe use some smaller cards that we could put in a single row instead of always using 1 card per row.

Reply:
Just for the record, I've been used my forked version of the dashboard that contains this code ( marcosdiez/dashboard:v2.3.1j ) and I personally thing it's quite good as is.

Some objects are created by replica sets, some are created by daemon sets, some by cron jobs, some by stateful sets and this tiny PR covers it all. If I need more info I just go and click (or middle click) on the `Owner` label. 

Maybe we should consider merging it as is.

Reply:
@floreks ^^^^

Reply:
Having this kind of generic resource owner information and creating internal links based on name/kind is not a good idea since we do not support every potential resource that might be created this way and it will result in dead/not working links inside the Dashboard.

We should support this explicitly on a per resource basis.

Reply:
@floreks you are right. I just added the code. I took the liberty of copying what you did the events file.